cpp/velox/memory/VeloxMemoryManager.cc

```diff
@@ -208,7 +215,13 @@ int64_t shrinkVeloxMemoryPool(velox::memory::MemoryManager* mm, velox::memory::M
 } // namespace
 
 const MemoryUsageStats VeloxMemoryManager::collectMemoryUsageStats() const {
-  return collectMemoryUsageStatsInternal(veloxAggregatePool_.get());
+  MemoryUsageStats stats;
+  stats.set_current(listener_->currentBytes());
+  stats.set_peak(listener_->peakBytes());
+  stats.mutable_children()->emplace("gluten_allocator", collectGlutenAllocatorMemoryUsageStats(glutenAlloc_.get()));
```

Comment:
Nit: Can it be better to use name `"gluten::MemoryAllocator"`? We have so many `allocator` definitions so it's better to use a very specific identifier to distinguish with others.

Reply:
ok.